query-connector/src/app/query/components/SearchForm.tsx

```diff
@@ -79,6 +79,10 @@ const SearchForm: React.FC<SearchFormProps> = ({
     [setUseCase],
   );
 
+  const nameRegex = "^[A-Za-z\u00C0-\u024F\u1E00-\u1EFF\\-'. ]+$";
```

Comment:
Is this regex based on the USWDS standard? I'm wondering what your thoughts are on aligning more closely with the [FHIR r4 standard for strings](https://www.hl7.org/fhir/r4/datatypes.html), which has a more permissive regex, `[ \r\n\t\S]+`. 

Of course, the fhir regex is  "informative, not normative" so it seems we have some leeway with how we implement.

Reply:
USWDS specifically calls out `hyphens, apostrophes, and blank spaces` as well as `"a rich array of diacritics, accents, and alternative characters"`, but then elsewhere advises `"Don’t restrict characters."` 😅 I think I'll leave it as-is for now but definitely happy to revisit if we decide we want to align more closely with r4!